evervault/__init__.py

```diff
@@ -35,8 +40,19 @@ def init(
     _retry = retry
     _curve = curve
 
-    if intercept:
-        __client().relay(ignore_domains)
+    if intercept or len(ignore_domains) > 0:
+        print(
+            "The `intercept` and `ignore_domains` config options in Evervault SDK are deprecated and slated for removal"
+        )
+        print("Please switch to the equivalent `decryption_domains config option.")
```

Comment:
Don't think it's accurate to say equivalent here, they're closer to inverse 
```suggestion
        print("Please switch to the `decryption_domains config option.")
```